packages/ra-ui-materialui/package.json

```diff
@@ -26,9 +26,9 @@
         "watch": "tsc --outDir dist/esm --module es2015 --watch"
     },
     "devDependencies": {
-        "@mui/icons-material": "^5.15.20",
-        "@mui/material": "^5.15.20",
-        "@mui/utils": "^5.15.20",
+        "@mui/icons-material": "^5.16.12",
+        "@mui/material": "^5.16.12",
+        "@mui/utils": "^5.16.12",
```

Comment:
You did not update the peerDeps in this file. Is it intended?

Reply:
Done